src/WarcraftLegacies.Source/Setup/FactionSetup/ScourgeSetup.cs

```diff
@@ -120,7 +120,10 @@ public static void Setup()
       Scourge.AddPower(visionPower);
       
       Scourge.AddGoldMine(PreplacedUnitSystem.GetUnit(FourCC("ngol"), new Point(-4939, 18803)));
-      
+
+      // Mechanics
```

Comment:
Let's see if we can get away with removing this comment. In my experience they're not maintained properly and I think what you've written is clear without it.